Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -855,7 +855,9 @@ extension ChatViewController {
             return UICollectionViewCell()
         }
 
-        cell.labelTitle.text = obj.timestamp.formatted("MMM dd, YYYY")
+        let dateFormatter = DateFormatter()
```

Comment:
Maybe let's create a singleton/static class that will hold formatter (one per format) and that will be responsible for formatting date? It's good practice to not unnecessarily create and destroy formatters that you're likely to need again especially in the cell's creation process.

Reply:
Yeah, you're right